specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -7773,6 +7773,15 @@
       },
       "description": "Specifies the storage settings for the virtual machine disks."
     },
+    "SecurityProfile": {
+        "properties": {
+            "encryptionAtHost": {
```

Comment:
>encryptionAtHost [](start = 13, length = 16)

Looks like this is a new writable property being added. I need following info
1. Will this break old SDK's for this API version? Do you've telemetry/data to prove?
2. Can older SDK cause overwrite of this parameter value while doing GET/PUT?

Reply:
Hi Chirag,
1.  This change will not break for any old SDKs as this is a nullable property and SDKs using the the api version wont send this property.  Also once the swagger changes are merged, we will be generating the SDK changes with this property .
2. The older SDKs as mentioned above will be sending us this as null which is handled in our code and GET also will set this property as null . So there wont be any breaking changes.

Reply:
Ok the scenario is 
1. Someone from portal created VM with encryptionAtHost settings configured.
2. OLD SDK based code would GET and will not recieve this field.
3. OLD SDK based code would PUT and nullify this field. Corrrect?
This is data loss and would change the behavior of the object.

Reply:
Hi Chirag, 
In this Scenario, when the customer used old SDK for the PUT operation, the property will be null and our code will ignore this null property and dont update any settings for the encryptionathost. So the user will still retain the encryptionathost settings after the PUT request with old SDK.